Site/webapp/wdkCustomization/js/client/components/Srt.tsx

```diff
@@ -84,37 +99,83 @@ const SUPPORTED_RECORD_CLASS_CONFIGS: InitialSrtFormConfig[] = [
     recordClassUrlSegment: 'est',
     display: 'EST IDs',
     ReporterForm: FastaGenomicSequenceReporterForm,
-    initialReporterFormState: {
+    defaultReporterFormState: {
       ...FastaGenomicSequenceReporterForm.getInitialState().formState,
       revComp: false,
       end: 200
     },
     makeInitialIdsState: () => '',
+    storedIdsState: undefined,
+    storedFormState: undefined,
     formActionUrl: '/cgi-bin/estSrt'
   },
   {
     recordClassUrlSegment: 'popsetSequence',
     display: 'Popset Isolate IDs',
     ReporterForm: FastaGenomicSequenceReporterForm,
-    initialReporterFormState: {
+    defaultReporterFormState: {
       ...FastaGenomicSequenceReporterForm.getInitialState().formState,
       revComp: false,
       end: 200
     },
     makeInitialIdsState: () => '',
+    storedIdsState: undefined,
+    storedFormState: undefined,
     formActionUrl: '/cgi-bin/isolateSrt'
   }
 ];
 
 export function Srt() {
+  const routeBase = useRouteMatch();
+  const { pathname } = useLocation();
   const compatibleSrtConfigs = useCompatibleSrtFormConfigs();
-  const [ selectedSrtForm, setSelectedSrtForm ] = useState<string>('');
+  const [selectedSrtForm, setSelectedSrtForm] = useState<string>('');
+  const [srtConfigsState, setSrtConfigsState] = useState<any>();
```

Comment:
Since you're only using `srtConfigsState` to store the id and form state, I think a name and type like `srtTabsState` would better convey how this state is getting used:

```suggestion
  const [srtTabsState, setSrtTabState] = useState<SrtTabsState>();
```

```typescript
type SrtTabsState = Record<
  string,
  {
    idsState: string;
    reporterFormState: Record<string, string | number | boolean>;
  }
>;
```